benchmark.py

```diff
@@ -176,10 +209,41 @@ def RunBenchmark(args, logfile):
   else:
     print "Instance deletion is skipped due to --no-create_delete."
 
+  ownip = StringIO.StringIO()
+  sh_utils.RunSSHCommand(args.username, args.vm_name,
```

Comment:
I feel it would probably make sense to have a persistent SSH connection (under pexpect) for this sort of thing. That way, you don't need to keep re-SSHing to the VM each time you need to do something. That would probably be a noticeable performance improvement. Please file an issue for this.